### PR TITLE
[9.0][MIG] purchase_request_to_procurement

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Backport of https://github.com/OCA/purchase-workflow/pull/339.

### CODE REVIEW

File: purchase_request_to_procurement/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    'name': 'Purchase Request To Procurement',
+    'summary': """
+        This module introduces the possiblity to
+        create procurement order from purchase request""",
+    'version': '9.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'www.acsone.eu',
+    'depends': [
+        # Odoo
```
Comment (MiquelRForgeFlow) [inline purchase_request_to_procurement/__openerp__.py:15]: <code># Odoo</code>→<code># Odoo:</code>

File: purchase_request_to_procurement/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        'procurement',
+        'stock',
+        # OCA purchase-workflow
```
Comment (mreficent) [inline purchase_request_to_procurement/__openerp__.py]: <code># OCA purchase-workflow</code>→<code># OCA purchase-workflow:</code>

File: purchase_request_to_procurement/views/purchase_request.xml [hunk 3]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+<data>
```
Comment (MiquelRForgeFlow) [inline purchase_request_to_procurement/views/purchase_request.xml]: <code>data</code> tag is not needed if you put the <code>odoo</code> tag. The same for the other views.

File: purchase_request_to_procurement/__openerp__.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply purchase_request_to_procurement/__openerp__.py:15]: really?

Comment (LoisRForgeFlow) [reply purchase_request_to_procurement/__openerp__.py:15]: I'll prefer to keep it as it was in v10. It's quite clear.

**Comment by LoisRForgeFlow** [comment]
Improvements forward-ported in https://github.com/OCA/purchase-workflow/pull/393

**Comment by LoisRForgeFlow** [comment]
@JonathanNEMRY Can you please review? Are you ok with the changes proposed?

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by JonathanNEMRY — APPROVED** [review]
:+1:

**Pull request merged**